Tests/AssociationMatrix_test.py

```diff
@@ -1,6 +1,29 @@
 from Snapper.AssociationMatrix import AssociationMatrix
 from Relationship.Relationship import Relationship
 from Relationship.Variable import Variable
+from unittest.mock import MagicMock
+import pytest
+
+
+@pytest.fixture()
+def rel_1():
```

Comment:
Should call these fixtures something that relates their contests. Perhaps `rel_25`, `rel_50`, and `rel_75`